ADONET/Program.cs

```diff
@@ -1,9 +1,8 @@
 ﻿using Microsoft.Data.Sqlite;
-using static System.Runtime.InteropServices.JavaScript.JSType;
 
 Console.WriteLine("Querying for blogs");
 string DbPath = "blogging.db";
-Console.WriteLine($"Database path: {DbPath}"); 
```

Comment:
The [Trailing Whitespace Visualizer](https://marketplace.visualstudio.com/items?itemName=MadsKristensen.TrailingWhitespace64) can automatically clean these up.

Reply:
Downloaded!